websites/M/MixMods/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "Pequeno",
+      "id": "323577179814494209"
+    },
+    "service": "MixMods",
+    "description": {
+      "en": "MixMods is a website of mods for GTA San Andreas and another games. Created by Junior_Djjr, is the biggest website of mods of America Latina",
+      "pt": "MixMods é um site de mods para GTA San Andreas e outros jogos. Criado por Junior_Djjr, é o maior site de mods da America Latina."
+    }, 
+    "url": "www.mixmods.com.br",
+    "version": "1.0.0",
+    "logo": "https://imgur.com/7YvTrtD",
+    "thumbnail": "https://imgur.com/x17ShNm",
```

Comment:
Please use the **DIRECT** link of the image.

Reply:
Done